File: gimpi/23-24/k1/02Poeni/2prijemni2.cpp
#include <bits/stdc++.h>
using namespace std;

int main() { 
    int n; cin >> n;
    int bodovi = 0;
    for (int i = 0; i < n; i++) {
        string proba, resenje;
        cin >> proba >> resenje;
        if ( proba == resenje ) bodovi += 4;
        else if ( proba != "ooooo" ) bodovi -= 1;     }
    cout << bodovi << '\n' << 4 * n << '\n' << -n << '\n';

}

File: gimpi/23-24/k1/02Poeni/2prijemni2.py
n = int(input())
bodovi = 0
for i in range(n):
    proba = input()
    resenje = input()
    if proba == resenje:
        bodovi += 4
    elif proba != "ooooo":
        bodovi -= 1
    # ako je proba ooooo, ne menjaju se bodovi

print(bodovi)
print(4 * n)
print(-1 * n)